hieradata/node/auxtel-mcm.cp.lsst.org.yaml

```diff
@@ -1,4 +1,7 @@
 ---
+classes:
+  - "profile::ccs::tomcat"
```

Comment:
Class inclusion is not allowed at the node layer.  It results in unexpected behavior when changing the role assigned to the node.  This should have failed CI: https://github.com/lsst-it/lsst-control/blob/production/spec/unit/hiera_class_inclusion_spec.rb but something seems to have gone wrong with that test.

Reply:
I've changed it to add the class via the role, not the node.